CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # [3.2.1](https://github.com/phalcon/cphalcon/releases/tag/v3.2.1) (2017-XX-XX)
 - Fixed inconsistent behaviour of `Phalcon\Config::merge` across minor version of PHP7 [#12779](https://github.com/phalcon/cphalcon/issues/12779)
 - Fixed visibility of `Phalcon\Mvc\Model\Query\Builder::{_conditionNotIn,_conditionIn,_conditionNotBetween,_conditionBetween}` to allow 3rd party libraries extend it
+- Fixed `Phalcon\Http\Response` method `getStatusCode()` to return (int) HTTP code only, instead of full string [#12895]
```

Comment:
`Phalcon\Http\Response::getStatusCode` and [#12895](https://github.com/phalcon/cphalcon/issues/12895)(missing url)